Http/Http.cs

```diff
@@ -0,0 +1,352 @@
+		}
+
+		public void Send(UnityWebRequest unityWebRequest, Action<UnityWebRequest> onSuccess = null,
+			Action<UnityWebRequest> onError = null, Action<UnityWebRequest> onNetworkError = null)
+		{
+			StartCoroutine(SendCoroutine(unityWebRequest, onSuccess, onError, onNetworkError));
+		}
+
+		public void Send(UnityWebRequest unityWebRequest, Action onSuccess = null,
+			Action onError = null, Action onNetworkError = null)
+		{
+			StartCoroutine(SendCoroutine(unityWebRequest, onSuccess, onError, onNetworkError));
+		}
+
+		public void Send(UnityWebRequest unityWebRequest, Action<HttpResponse> onSuccess = null,
+			Action<HttpResponse> onError = null)
+		{
+			StartCoroutine(SendCoroutine(unityWebRequest, onSuccess, onError));
+		}
+
+		#endregion
+
+		#region Send HttpRequest methods
+
+		private static IEnumerator SendCoroutine(HttpRequest request, Action<HttpResponse> onSuccess,
+			Action<HttpResponse> onError)
+		{
+			var unityWebRequest = request.UnityWebRequest;
+			yield return unityWebRequest.SendWebRequest();
+			var response = new HttpResponse(unityWebRequest);
+
+			if (onError != null && (unityWebRequest.isNetworkError || unityWebRequest.isHttpError))
+			{
+				onError.Invoke(response);
+			}
+			else if (onSuccess != null)
+			{
+				onSuccess.Invoke(response);
+			}
+		}
+
+		#endregion
+
+		#region Send UnityWebRequest methods
+
+		private static IEnumerator SendCoroutine(UnityWebRequest unityWebRequest,
+			Action onSuccess = null, Action onError = null, Action onNetworkError = null)
+		{
+			yield return unityWebRequest.SendWebRequest();
+			if (unityWebRequest.isNetworkError)
+			{
+				if (onNetworkError != null)
+				{
+					onNetworkError.Invoke();
+				}
+			}
+			else if (unityWebRequest.isHttpError)
+			{
+				if (onError != null)
+				{
+					onError.Invoke();
+				}
+			}
+			else
+			{
+				if (onSuccess != null)
+				{
+					onSuccess.Invoke();
+				}
+			}
+		}
+
+		private static IEnumerator SendCoroutine(UnityWebRequest unityWebRequest,
+			Action<UnityWebRequest> onSuccess = null,
+			Action<UnityWebRequest> onError = null, Action<UnityWebRequest> onNetworkError = null)
+		{
+			yield return unityWebRequest.SendWebRequest();
+			if (unityWebRequest.isNetworkError)
+			{
+				if (onNetworkError != null)
+				{
+					onNetworkError.Invoke(unityWebRequest);
+				}
+			}
+			else if (unityWebRequest.isHttpError)
+			{
+				if (onError != null)
+				{
+					onError.Invoke(unityWebRequest);
+				}
+			}
+			else
+			{
+				if (onSuccess != null)
+				{
+					onSuccess.Invoke(unityWebRequest);
+				}
+			}
+		}
+
+		private static IEnumerator SendCoroutine(UnityWebRequest request,
+			Action<HttpResponse> onSuccess,
+			Action<HttpResponse> onError)
+		{
+			yield return request.SendWebRequest();
+			var response = new HttpResponse(request);
+
+			if (onError != null && (request.isNetworkError || request.isHttpError))
+			{
+				onError.Invoke(response);
+			}
+			else if (onSuccess != null)
```

Comment:
Same here, this could get called incorrectly if `(request.isNetworkError || request.isHttpError)` is true but `onError` is null